Solving Issue #3155 (#3418)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-channels-overview.md

## Org-wide teams

If your organization has no more than 5,000 users, you can create an org-wide team. Org-wide teams provide an automatic way for everyone in an organization to be a part of a single team for collaboration. For more information, including best practices for creating and managing an org-wide team, see [Create an org-wide team in Microsoft Teams](create-an-org-wide-team.md).

# How to hide a channel

> In case that you want to hide a channel inside a teams, right click and select "hide" as shown in the next picture:

Reviewer: ```suggestion
If you want to hide a channel inside a team, right click and select **Hide** as shown in the following illustration.
```

>   > ![How to leave a Team](media/HideAChannel.png)

Reviewer: ```suggestion
  > ![How to leave a team](media/HideAChannel.png)
```

> # How to leave a Teams

Reviewer: ```suggestion
# How to leave a team
```

> In case that you are willing to get out of a teams, you can right click the channel and then select "Leave the Team" as shown in the next picture:

Reviewer: ```suggestion
If you want to leave a team, right click the channel and then select **Leave the Team** as shown in the following illustration.
```

>   > ![How to leave a Team](media/LeaveATeam.png)

Reviewer: The link to this graphic does not seem to work.
